### ISSUE TITLE
Problem with update_dataset

### ISSUE THREAD

**Issue by mahmoudelkasabi** [RamiKrispin/coronavirus#50]
Hi Rami,

I upgraded to R 4.0.0 recently. Since then I'm getting the error below with update_dataset(). I removed the coronvirus package and re-installed it several times but no luck.

`> update_dataset()
Updates are available on the coronavirus Dev version, do you want to update? n/YY
Downloading GitHub repo RamiKrispin/coronavirus@master
√  checking for file 'C:\Users\31672\AppData\Local\Temp\Rtmp6dHYtb\remotesef875d26408\RamiKrispin-coronavirus-e77e858/DESCRIPTION' (405ms)
-  preparing 'coronavirus': (2s)
√  checking DESCRIPTION meta-information ...
-  checking for LF line-endings in source and make files and shell scripts
-  checking for empty or unneeded directories
-  building 'coronavirus_0.2.0.tar.gz'
   
Caught an warning!
<simpleWarning: package ‘coronavirus’ is in use and will not be installed>`

**Comment by dtdibaba** [comment]
You may have to restart R or remove it (remove.packages("coronavirus") ) and re-install.

**Comment by mahmoudelkasabi** [comment]
Thanks. I already did that but no luck.

**Comment by dtdibaba** [comment]
Does it keep showing the last warning?

**Comment by mahmoudelkasabi** [comment]
Yes.

**Comment by RamiKrispin** [comment]
Could you please try to run the following and let me know what is the output you are getting?

```
devtools::install_github("RamiKrispin/coronavirus",
                                   upgrade = "never",
                                   ref = "master")
```

**Comment by mahmoudelkasabi** [comment]
Great. This one works fine. Thanks a lot.

**Comment by RamiKrispin** [comment]
Generally, this is the function that function uses to install the data. Can you please try to run the function again?

**Comment by mahmoudelkasabi** [comment]
the update_dataset() is not working yet. it gives the same error. Also, once I run update_dataset() and then try to run the install_github() I get the error below 
`Error: Failed to install 'coronavirus' from GitHub:
  (converted from warning) package ‘coronavirus’ is in use and will not be installed`
To make it work again, I use the following
`remove.packages("coronavirus")
install.packages("coronavirus")
devtools::install_github("RamiKrispin/coronavirus",
                         upgrade = "never",
                         ref = "master")`

**Comment by RamiKrispin** [comment]
What is the output of the `sessionInfo()`?

**Comment by mahmoudelkasabi** [comment]
`
> sessionInfo()
R version 4.0.0 (2020-04-24)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 7 x64 (build 7601) Service Pack 1

Matrix products: default

locale:
[1] LC_COLLATE=English_United States.1252  LC_CTYPE=English_United States.1252   
[3] LC_MONETARY=English_United States.1252 LC_NUMERIC=C                          
[5] LC_TIME=English_United States.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] coronavirus_0.2.0

loaded via a namespace (and not attached):
 [1] rstudioapi_0.11   magrittr_1.5      usethis_1.6.1     devtools_2.3.0    pkgload_1.0.2     R6_2.4.1         
 [7] rlang_0.4.6       fansi_0.4.1       tools_4.0.0       pkgbuild_1.0.8    packrat_0.5.0     sessioninfo_1.1.1
[13] cli_2.0.2         withr_2.2.0       ellipsis_0.3.0    remotes_2.1.1     assertthat_0.2.1  digest_0.6.25    
[19] rprojroot_1.3-2   crayon_1.3.4      processx_3.4.2    callr_3.4.3       fs_1.4.1          ps_1.3.3         
[25] curl_4.3          testthat_2.3.2    memoise_1.1.0     glue_1.4.1        compiler_4.0.0    desc_1.2.0       
[31] backports_1.1.7   prettyunits_1.1.1
> 
`

**Comment by RamiKrispin** [comment]
Can you run the following on a new session and check if you are getting any errors or warnings?

```
library(coronavirus)
data(coronavirus)

devtools::install_github("RamiKrispin/coronavirus",
                                   upgrade = "never",
                                   ref = "master")

          if ("coronavirus" %in% names(utils::sessionInfo()$otherPkgs)) { 
            detach(package:coronavirus, unload = TRUE)
            library(coronavirus)
           }
```

**Comment by mahmoudelkasabi** [comment]
It gives me the error below

`
Downloading GitHub repo RamiKrispin/coronavirus@master
√  checking for file 'C:\Users\31672\AppData\Local\Temp\RtmpEnY4il\remotes292c4d9f71c9\RamiKrispin-coronavirus-80b4520/DESCRIPTION' (474ms)
-  preparing 'coronavirus': (1.8s)
√  checking DESCRIPTION meta-information ... 
-  checking for LF line-endings in source and make files and shell scripts
-  checking for empty or unneeded directories
-  building 'coronavirus_0.2.0.tar.gz'
   
Error: Failed to install 'coronavirus' from GitHub:
  (converted from warning) package ‘coronavirus’ is in use and will not be installed
`

**Comment by RamiKrispin** [comment]
I tested it with R4.0.0 using a docker container and I did not have any issue. I am wondering if this error occurs when installing other packages with `devtools::install_github`?

**Comment by jaemariebarnes** [comment]
I am having problems as well (data only goes to 12 May:
I had to download library glue and it says  need Rtools, but when I try downloading I get:
Warning in install.packages :
  package ‘Rtools’ is not available (for R version 3.6.3)
Warning in install.packages :
  unable to access index for repository http://cran.r-project.org/bin/windows/Rtools/bin/windows/contrib/3.6:
  cannot open URL 'http://cran.r-project.org/bin/windows/Rtools/bin/windows/contrib/3.6/PACKAGES'
Warning message:
In library(package, lib.loc = lib.loc, character.only = TRUE, logical.return = TRUE,  :
  there is no package called ‘Rtools’
> devtools::install_github("RamiKrispin/coronavirus",
+                          upgrade = "never",
+                          ref = "master")
WARNING: Rtools is required to build R packages, but is not currently installed.

Please download and install Rtools custom from http://cran.r-project.org/bin/windows/Rtools/.
Skipping install of 'coronavirus' from a github remote, the SHA1 (b739afda) has not changed since last install.
  Use `force = TRUE` to force installation
 library(coronavirus)
> data(coronavirus)
> 
> devtools::install_github("RamiKrispin/coronavirus",
+                          upgrade = "never",
+                          ref = "master",force=TRUE)
WARNING: Rtools is required to build R packages, but is not currently installed.

Please download and install Rtools custom from http://cran.r-project.org/bin/windows/Rtools/.
Downloading GitHub repo RamiKrispin/coronavirus@master
WARNING: Rtools is required to build R packages, but is not currently installed.

Please download and install Rtools custom from http://cran.r-project.org/bin/windows/Rtools/.
√  checking for file 'C:\Users\massarojo\AppData\Local\Temp\Rtmp2tV3Lk\remotes6ae422b53422\RamiKrispin-coronavirus-b739afd/DESCRIPTION' ...

**Comment by jaemariebarnes** [comment]
Never mind, got it to work.
